src/classes/WildebeestModel.ts

```diff
@@ -210,15 +210,21 @@ export default class WildebeestModel<TModels extends ModelMap> extends Model {
   /**
    * Get the model definition by merging the default definition and definition
    *
-   * TODO improve return type to use this
-   *
+   * @param this -This model
    * @param mergeDefaults - The defaults to merge with the existing defaults
    * @returns The merged default definitions
    */
   public static mergeDefaultDefinitions<
-    TNewDefaults extends Partial<ModelDefinition<StringKeys<ModelMap>>>,
-    M extends any
-  >(mergeDefaults: TNewDefaults): TNewDefaults & M['definitionDefaults'] {
+    TNewDefaults extends Partial<ModelDefinition<any>>, // StringKeys<ModelMap>
```

Comment:
This comment // StringKeys does not make sense to me, but is copied over from the code in Main (linked in PR description).